src/auth/aws.test.js

```diff
@@ -303,7 +310,7 @@ test('processHeaders content header is correctly hashed', (t) => {
 
     const headers = _.processHeaders(req, 'body-content');
     const contentHeader = headers.find(el => el.name === 'x-amz-content-sha256');
-    t.equal(contentHeader.value, sha256('body-content').toString());
+    t.equal(contentHeader.value, _.toSHA256('body-content').toString());
```

Comment:
now we have our wrapper function we should stub the actual method out :) 

I tried before but without the wrapper I couldn't get it to work.